src/components/loader/loader.test.tsx

```diff
@@ -30,13 +44,21 @@ test("throws a deprecation warning if the 'aria-label' prop is set", () => {
   loggerSpy.mockRestore();
 });
 
-test("when the user disallows animations or their preference cannot be determined, alternative loading text is rendered", () => {
+test("does not render anything when `reduceMotion` is undefined", () => {
+  mockUseMediaQuery.mockReturnValueOnce(undefined);
+
+  render(<Loader />);
+
+  expect(screen.queryByTestId("loader")).not.toBeInTheDocument();
+});
+
+test("when the user disallows animations or their preference cannot be determinded, alternative loading text is rendered", () => {
   render(<Loader />);
 
   expect(screen.getByText("Loading")).toBeVisible();
 });
 
-test("when the user disallows animations or their preference cannot be determined, the provided `aria-label` is rendered", () => {
+test("when the user disallows animations or their preference cannot be determinded, the provided `aria-label` is rendered", () => {
```

Comment:
suggestion: fix typo of determined

```suggestion
test("when the user disallows animations or their preference cannot be determined, the provided `aria-label` is rendered", () => {
```